src/platform_impl/windows/event_loop.rs

```diff
@@ -1063,14 +1095,14 @@ unsafe fn public_window_callback_inner<T: 'static>(
       result = ProcResult::DefSubclassProc;
     }
 
-    winuser::WM_SIZE => {
+    win32wm::WM_SIZE => {
       use crate::event::WindowEvent::Resized;
-      let w = LOWORD(lparam as DWORD) as u32;
-      let h = HIWORD(lparam as DWORD) as u32;
+      let w = lparam.0 as u32 & 0xFFFF;
+      let h = (lparam.0 as u32 & 0xFFFF_0000) >> 16;
```

Comment:
Sure, that and the equivalent to `GET_X_LPARAM`/`GET_Y_LPARAM`, which should use `i16` instead of `u16`.

Reply:
yeah that would be nice, Also do you have any idea why `windows-rs` doesn't have these functions ?

Reply:
They might make sense to add by hand to `windows-rs` itself, but the reason they don't exist in the generated bindings is that they're macros rather than functions.